services/datamanager/collection/collection.go

```diff
@@ -194,6 +196,12 @@ func GetCollections(params GetCollectionsParams) (results ReadCollections, err e
 			return
 		}
 
+		// Update count of remaining collections
+		remainingCollections = collections.ItemsAvailable - params.BatchSize - sdkParams["offset"].(int)
```

Comment:
Ok, I'll set `batchCollections := len(collections.Items)` once and use that throughout.